erp_discarded components still in ERP_final: set()
 erp_discarded components still in ERP_LIST1_final: {'153931', '2111779', '2109429', '2112146', '2109344', '2109345'}
Unique RM Specs after discarding off specs 337
77 rows discarded from ERP as they are irrelevant materials
34 rows discarded from TRU as they are irrelevant materials
############################################################
Common number of codes in both sources:  73
Ingest Data took 226.57260274887085 seconds
------------------------------------------------------------
Format raw material took 0.004874467849731445 seconds
erp part desc nulls 0
tru spec desc nulls:  0
processing erp
processing tru
Process text description took 0.15341830253601074 seconds
processing tru trade names
Process text description took 0.16116046905517578 seconds
Cleaning up all outstanding Run operations, waiting 300.0 seconds
1 items cleaning up...
Cleanup took 0.10435724258422852 seconds
Traceback (most recent call last):
  File "cross_hound/matcher.py", line 1463, in <module>
    main(input_dir=args.get('input_dir'),
  File "cross_hound/matcher.py", line 1288, in main
    matched, unmatched = full_matching_routine(data_version, db_style, device,
  File "cross_hound/matcher.py", line 760, in full_matching_routine
    erp = create_features(erp, 'erp')
  File "/mnt/azureml/cr/j/3d9feaaa1bbe43348e8dbffff8930650/exe/wd/cross_hound/feature_engineering.py", line 69, in create_features
    processed_desc_tmp = df_copy[f'processed_desc_{source}'].str.replace(pattern, '', regex=True)
  File "/opt/conda/envs/ptca/lib/python3.8/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
  File "/opt/conda/envs/ptca/lib/python3.8/site-packages/pandas/core/accessor.py", line 224, in __get__
    accessor_obj = self._accessor(obj)
  File "/opt/conda/envs/ptca/lib/python3.8/site-packages/pandas/core/strings/accessor.py", line 180, in __init__
    self._inferred_dtype = self._validate(data)
  File "/opt/conda/envs/ptca/lib/python3.8/site-packages/pandas/core/strings/accessor.py", line 234, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!
